data/human/transport missions.txt

```diff
@@ -3582,3 +3582,743 @@ mission "Hauler VI cargo prototype 3"
+				`	"I would love to."`
+				`	"Why not."`
+				`	"Sorry, I have more pressing matters at hand."`
+					goto oh
+			`	"Great," she says. "Go and bring your ship to this hanger. I will have the crew get everything ready to load."`
+				accept
+			label oh
+			`	"Oh, I'm surprised you would turn down so much money for such little work and a chance to meet the artists. But it's your call," she says. "The exhibit is closed. I'm going to have to ask you to leave."`
+				decline
+	on visit
+		dialog `You arrive on <planet>, but realize that your escort carrying the exposition has not arrived yet. Better depart and wait for your escorts to enter the system.`
+	on complete
+		payment 80000
+		log `Saw an impressive exposition on the Hope evaluation and ended up with a job taking the whole thing to Moonshake.`
+		conversation
+			`	You guide your ship down to the indicated hanger where there is a motley looking crew waiting to unload your ship. One of them hands you a note. It reads: "Meet me at the Common Grounds cafe in the spaceport if you're interested in doing a project with us. Make sure you're not followed. -Adelita Zero"`
+
+mission "Journalism 2"
+	description "Participate in the guerrilla filming of a Syndicate sweatshop on <destination>."
+	name "Film Sweatshop"
+	source "Moonshake"
+	destination "Maker"
+	passengers 3
+	cargo "stealth drones" 1
+	to offer
+		has "Journalism 1: done"
+
+	on offer
+		conversation
+			`You find the Common Grounds cafe. It has a funky vibe and it's quite busy. You don't seem to be recognized by anyone so you take a seat at a booth and order something to eat. Before your meal arrives a woman, maybe in her late thirties with dark hair, sits down across from you and hails the server. She cheerfully orders a smoothie with a long list of ingredients, many of which you have never heard of, or can even pronounce.`
+			`	"Sorry for the cloak and dagger bit," she says to you finally. "We have enemies in high places. So, Captain <last> "What did you think of the exposition?"`
+			choice
+				`	"I found it quite moving."`
+				`	"It was OK."`
+					goto wasok
+				`	"It was too political for me."`
+					goto political
+			`	"I'm glad you liked it," she says smiling.`
+				goto inorout
+			label wasok
+			`	"Not exactly a thrilling review, but you're entitled to your opinion," she says smiling.`
+				goto inorout
+			label political
+			`	"Oh. Well unfortunately for many, everything is political. It was political when the Republic tried to block the evacuation. And it was political when the Navy did it without permission. And the same goes for every single act of oppression and liberation since the dawn of humanity. Our oppressors are very political. Maybe you have the hope of being one some day; wealthy, authoritative. Or maybe you just don't care. Either way, you don't sound like a good fit for this project. But thanks for transporting the installation," she says with a smile. She gets her smoothie to-go and walks out.`
```

Comment:
```suggestion
			`	"Oh. Well, unfortunately for many, everything is political. It was political when the Republic tried to block the evacuation. It was political when the Navy did it without permission. And the same goes for every single act of oppression and liberation since the dawn of humanity. Our oppressors are very political. Maybe you have the hope of being one some day: wealthy, authoritative. Or maybe you just don't care. Either way, you don't sound like a good fit for this project. But thanks for transporting the installation," she says with a smile. She gets her smoothie to-go and walks out.`
```